### ISSUE TITLE
Search issues with punctuation, colons etc

### ISSUE THREAD

**Issue by tejapoth** [GamerKingFaiz/shouldiplay#8]
Hi 
I’m having issues with search with punctuation and colons. Most of the time it works, but some specific cases it doesn’t work.
Here are the examples:

- the é in Pokémon causing issues and doesn’t return any results. But without é its working. - [Pokémon Omega Ruby and Alpha Sapphire](https://shouldiplaythis.com/?page=1&search=Pok%C3%A9mon%20Omega%20Ruby%20and%20Alpha%20Sapphire)
- colon >> **:** and hyphen >> **-** , don’t work for searching - [Shantae: Half-Genie Hero](https://shouldiplaythis.com/?page=1&search=Shantae:%20Half-Genie%20Hero)
- semicolon >> **;** - [Hyperdimension Neptunia Re;Birth 1](https://shouldiplaythis.com/?page=1&search=Hyperdimension%20Neptunia%20Re%3BBirth%201)
- this is a good title for you to test against.  - [JoJo's Bizarre Adventure: All-Star Battle](https://shouldiplaythis.com/?page=1&search=JoJo%27s%20Bizarre%20Adventure:%20All-Star%20Battle)

In summary, please fix french accents i.e Pok**é**mon and **:** and **-** and **;**

thanks for your work on this extremely helpful site
